app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -410,23 +392,6 @@ class MainActivity : DrawerActivity() {
         }
     }
 
-    private fun destinationPreviewCustomization() {
-        bindSwitch(
-            switch = menuBinding.setCustomDestinationPreview,
-            state = setCustomDestination,
```

Comment:
Please remove `setCustomDestination` flow and `setCustomDestinationPreview` switch in the layout

Reply:
Done. Thanks.